Task1_Library/Library.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Task1_Library
+{
+    public class Library
+    {
+        public static string OutPutLogic(string input)
+        {
+            var time = DateTime.Now.ToString();
+            return time + " ,Hello, " + input;
```

Comment:
Change to format string with interpolation https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated